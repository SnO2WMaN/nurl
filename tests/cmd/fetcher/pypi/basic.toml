args = ["https://pypi.org/project/requests", "2.20.0"]
