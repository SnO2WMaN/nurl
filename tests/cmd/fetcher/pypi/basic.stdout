fetchPypi {
  pname = "requests";
  version = "2.20.0";
  hash = "sha256-mdz9qusXyvblJvMrant4BGFRKrPx2ZIYeAFpTLpCdww=";
}